requirements.txt~

```diff
@@ -0,0 +1,16 @@
+Flask==0.10.1
+Jinja2==2.7.3
+MarkupSafe==0.23
+PennSDK==1.1
+Werkzeug==0.9.6
+argparse==1.2.1
+beautifulsoup4==4.3.2
+html5lib==0.999
+itsdangerous==0.24
+penncoursereview==0.0.3
+pymongo==2.7.2
+pytz==2014.7
+redis==2.10.3
+requests==2.4.3
+six==1.9.0
+wsgiref==0.1.2
```

Comment:
Amend commit to remove this and fix your global gitignore to ignore these kinds of files globally.